src/services/analytics/tx-tracking.ts

```diff
@@ -63,12 +62,14 @@ export const getTransactionTrackingType = async (chainId: string, txId: string):
     }
 
     if (details.safeAppInfo) {
-      return isWalletConnectSafeApp(details.safeAppInfo.url) ? TX_TYPES.walletconnect : details.safeAppInfo.url
+      return details.safeAppInfo.url
     }
 
     if (isMultiSendTxInfo(txInfo)) {
       return TX_TYPES.batch
     }
+
+    return TX_TYPES.walletconnect
```

Comment:
Is it safe to assume wc here as the fallback? Previously it would fallback to custom which is more generic.

Reply:
Not ideal but I think it's OK to assume that if it's not an app, it must be WalletConnect. I don't think there's any other way to sign or execute a tx from our UI.